app/models/data_center.rb

```diff
@@ -47,6 +47,9 @@
 #  index_data_centers_on_data_center_key  (data_center_key)
 #
 class DataCenter < ApplicationRecord
+  # Data Center for validators not assigned anywhere (mostly due to lack of validator_ip) 
+  UNKNOWN_DATA_CENTER_KEY = "0--Unknown"
```

Comment:
I would just call it unknown - it should look better in views

Reply:
It's for the ASN view to have number